src/components/fx/layout_attributes.js

```diff
@@ -57,7 +57,7 @@ module.exports = {
     hovermode: {
         valType: 'enumerated',
         role: 'info',
-        values: ['x', 'y', 'closest', false],
+        values: ['x', 'y', 'closest', false, 'x unified', 'y unified'],
```

Comment:
@antoinerg would you please update the description of `hovermode` in respect to these two new attributes?

Reply:
@nicolaskruchten could you tackle this one (ie. updating the description of `hovermode` to include `x unified` and `y unified`) ?

Reply:
ping @nicolaskruchten

Reply:
Please insert, before the existing paragraph, something like:

If *closest*, a single hoverlabel will appear for the closest point within the `hoverdistance`.
If *x* (or *y*), multiple hoverlabels will appear for multiple points at the closest x- (or y-) coordinate within the `hoverdistance`, with the caveat that no more than one hoverlabel will appear per trace.
If *x unified* (or *y unified*), a single hoverlabel will appear multiple points at the closest x- (or y-) coordinate within the `hoverdistance`, with the caveat that no more than one hoverlabel will appear per trace. In this mode, `spikelines` are enabled by default perpendicular to the specified axis.
If *false*, hover interactions are disabled.

Edit to taste and/or to correct inaccuracies :)